src/web/crate_details.rs

```diff
@@ -610,4 +611,95 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn test_updating_owners() {
+        wrapper(|env| {
+            let db = env.db();
+
+            env.fake_release()
+                .name("foo")
+                .version("0.0.1")
+                .add_owner(CrateOwner {
+                    login: "foobar".into(),
+                    avatar: "https://example.org/foobar".into(),
+                    name: "Foo Bar".into(),
+                    email: "foobar@example.org".into(),
+                })
+                .create()?;
+
+            let details = CrateDetails::new(&db.conn(), "foo", "0.0.1").unwrap();
+            assert_eq!(
+                details.owners,
+                vec![("foobar".into(), "https://example.org/foobar".into())]
+            );
+
+            // Adding a new owner, and changing details on an existing owner
+            env.fake_release()
+                .name("foo")
+                .version("0.0.2")
+                .add_owner(CrateOwner {
+                    login: "foobar".into(),
+                    avatar: "https://example.org/foobarv2".into(),
+                    name: "Foo Bar".into(),
+                    email: "foobar@example.org".into(),
+                })
+                .add_owner(CrateOwner {
+                    login: "barfoo".into(),
+                    avatar: "https://example.org/barfoo".into(),
+                    name: "Bar Foo".into(),
+                    email: "foobar@example.org".into(),
+                })
+                .create()?;
+
+            let details = CrateDetails::new(&db.conn(), "foo", "0.0.1").unwrap();
+            let mut owners = details.owners.clone();
+            owners.sort();
+            assert_eq!(
+                owners,
+                vec![
+                    ("barfoo".into(), "https://example.org/barfoo".into()),
+                    ("foobar".into(), "https://example.org/foobarv2".into())
+                ]
+            );
+
+            // Removing an existing owner
+            env.fake_release()
+                .name("foo")
+                .version("0.0.3")
+                .add_owner(CrateOwner {
+                    login: "barfoo".into(),
+                    avatar: "https://example.org/barfoo".into(),
+                    name: "Bar Foo".into(),
+                    email: "foobar@example.org".into(),
+                })
+                .create()?;
+
+            let details = CrateDetails::new(&db.conn(), "foo", "0.0.1").unwrap();
+            assert_eq!(
+                details.owners,
+                vec![("barfoo".into(), "https://example.org/barfoo".into())]
+            );
+
+            // Changing owner details on another of there crates applies too
```

Comment:
```suggestion
            // Changing owner details on another of their crates applies the change to both
```
and thank you for the test!